package.json

```diff
@@ -628,6 +634,7 @@
   "dependencies": {
     "@kubernetes/client-node": "^0.8.1",
     "byline": "^5.0.0",
+    "clipboardy": "^1.2.3",
```

Comment:
Why don't you use the VSCode API as mentioned by @dgolovin

Reply:
@sudhirverma same question here. VSCode already has API for accessing clipboard, no need to use additional modules.

Reply:
okay I will use the VSCode API for accessing clipboard